### ISSUE TITLE
Zigbee Door Window Sensor for Tuya ZM-CG206

### ISSUE THREAD

**Issue by dioneimiodutzki** [dresden-elektronik/deconz-rest-plugin#4878]
## Device

- Product name: Zigbee Door Window Sensor for Tuya 
- Manufacturer: Tuya (?)
- Model identifier: ZM-CG206
- Device type : 
  - Sensor

### Aditional Info

This is a chinese generic door/window sensor (open/close sensor) but somehow is not listed like that on Deconz. 

https://www.aliexpress.com/item/1005002348194512.html?spm=a2g0s.9042311.0.0.19be4c4duwG9If

## Screenshots

![image](https://user-images.githubusercontent.com/26262726/118324129-a2f30e80-b4cf-11eb-9a28-a6a4499c0151.png)

### Power Configuration

![image](https://user-images.githubusercontent.com/26262726/118324285-d2098000-b4cf-11eb-8a95-9b677454d102.png)

### IAS Zone

![image](https://user-images.githubusercontent.com/26262726/118324310-dc2b7e80-b4cf-11eb-9e27-46ea8c6a3a76.png)

**Comment by Smanar** [comment]
I think it s not correclty included, it miss some cluster and have "ffff".
Can be repaired naturaly after a time, else you can use command from the first circle to finish it.

**Comment by dioneimiodutzki** [comment]
> 
> 
> I think it s not correclty included, it miss some cluster and have "ffff".
> Can be repaired naturaly after a time, else you can use command from the first circle to finish it.

Sorry, I never hit an unsupported device before so I really don't know how to proceed. 
How can I try to send a command to finish it?

**Comment by Smanar** [comment]
Easy, when you select the first circle on the node title, you will have a popup with some commands. Just resend them (specially the descriptor ones), for that you just need to click on the square.
But you will probably need to wake up the device in same time (this issue happen generaly when the device go to sleep during the inclusion)

But it can be repaired itslef after a time.

**Comment by dioneimiodutzki** [comment]
Thanks! I was able to find the buttons and try, but to no luck, even waking up the sensor, the buttons have no effect. I tried then to repair the sensor (remove it on deconz and started the search on phoscon again, and the sensor appeared again the same way (ffff). I've bought a pack of 4 of those sensors and they are all with the same behaviour. Not sure if it will be fixable. Anything else I can try?
Again, thanks for the help.
